src/main/java/pl/coderstrust/accounting/controller/InvoiceController.java

```diff
@@ -35,10 +41,12 @@
 
   private InvoiceService invoiceService;
   private InvoiceValidator invoiceValidator;
+  private PdfService invoicePdf;
```

Comment:
Please rename to pdfService - current name is misleading

Reply:
done